RevenueCat/Scripts/Purchases.cs

```diff
@@ -8,31 +8,6 @@
 
 public partial class Purchases : MonoBehaviour
 {
-    public delegate void LogInFunc(CustomerInfo customerInfo, bool created, Error error);
-
-    public delegate void GetOfferingsFunc(Offerings offerings, Error error);
-
-    public delegate void CheckTrialOrIntroductoryPriceEligibilityFunc(Dictionary<string, IntroEligibility> products);
```

Comment:
I moved these closer to where they're actually used, so it's much easier to understand what they actually do